CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ are made before the last `apostrophe:modulesRegistered` handler has fired.
 If you need to call Apostrophe's `find()` methods at startup,
 it is best to wait for the `@apostrophecms/doc:beforeReplicate` event.
 * Allow `@` when a piece is a template and `/@` for page templates (doc-template-library module).
+* Adds a `doNotAddPrefix` option to frontend http util module in order to avoid adding a prefix to already-prefixed URLs when making frontend calls.
```

Comment:
Go with `prefix: false` instead. This is more consistent with our pattern elsewhere. We avoid options with negated names.